pom.xml

```diff
@@ -28,6 +32,10 @@
             <id>krmorse</id>
```

Comment:
delete this whole developers section... I don't think we want people to think we actually actively develop this anymore

Reply:
done